assets/scss/_home.scss

```diff
@@ -576,8 +576,38 @@
         }
     }
 
-    #complextopics .container {
-        margin-top: auto;
+    #complextopics {
+        @media (min-width: 1600px) {
+            max-height: 643px;
+        }
+    
+        .container {
+            margin-top: auto;
+        }
+
+        .section-img {
```

Comment:
Since the other images are wider, I think we should add some padding to the container here so the layout is similar. I think `padding-left: 5vw; padding-right: 5vw;` works well.